src/hotspot/share/classfile/javaClasses.hpp

```diff
@@ -180,10 +181,24 @@ class java_lang_String : AllStatic {
     return h;
   }
 
+  static unsigned int hash_code(const char *utf8_str, int utf8_len) {
+    unsigned int h = 0;
+    int num_unicode_points = UTF8::unicode_length(utf8_str, utf8_len);
```

Comment:
```suggestion
    int unicode_length = UTF8::unicode_length(utf8_str, utf8_len);
```
It is clearer to always use the same name for the local assigned from `UTF8::unicode_length`.